pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -283,6 +282,21 @@ func setReservation(disk *api.Disk) {
 	}
 }
 
+func setErrorPolicy(diskDevice *v1.Disk, disk *api.Disk) error {
+	if diskDevice.ErrorPolicy == "" {
+		log.Log.Infof("Setting default error policy of %s for disk %s", v1.DiskErrorPolicyStop, diskDevice.Name)
```

Comment:
Do I understand it right, that this will be printed for every disk and on each sync? Maybe we shall reconsider if the message is needed at all here (previously nothing was printed)? If so, then maybe use a debug log level?

Reply:
@vasiliy-ul yes you are right, otherwise it becomes too noisy